### PR TITLE
Automation Scripts

### PR DESCRIPTION
**Pull request by PayalChoksey17**

### CODE REVIEW

File: tests/GiriLife/GiriLife.prj [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Project>
+   <description></description>
+   <name>GiriLife</name>
```
Comment (brylie) [inline tests/GiriLife/GiriLife.prj]: The project name is GeriLife.

File: app/lib/accounts.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const userEmailSchema = require("./userEmailSchema");
 AccountsTemplates.configure({
   // Behavior
-  forbidClientAccountCreation: true,
+  forbidClientAccountCreation: false,
```
Comment (brylie) [inline app/lib/accounts.js]: We don't yet want to change this setting permanently.

File: tests/GiriLife/.gitignore.gitignore [hunk 3]
```diff
@@ -0,0 +1,11 @@
+*.log
```
Comment (brylie) [inline tests/GiriLife/.gitignore.gitignore]: What is .gitignore.gitignore?

**Comment by brylie** [comment]
Be sure to also add instructions for running these tests.

File: tests/GiriLife/.gitignore.gitignore (hunk 3 above)
Comment (PayalChoksey17) [reply tests/GiriLife/.gitignore.gitignore]: Changed the file name

File: app/lib/accounts.js (hunk 2 above)
Comment (PayalChoksey17) [reply app/lib/accounts.js]: Noted.

File: tests/GiriLife/GiriLife.prj (hunk 1 above)
Comment (PayalChoksey17) [reply tests/GiriLife/GiriLife.prj]: Noted.

**Comment by PayalChoksey17** [comment]
@brylie  will add Instruction file to run the test in the next pull .

File: app/lib/accounts.js (hunk 2 above)
Comment (brylie) [reply app/lib/accounts.js]: OK, go ahead and revert this change.

**Comment by brylie** [comment]
It looks like somehow a `wellbeing` submodule was added.

Perhaps we should just 're-do' this PR, so we don't change anything but adding the `tests` folder.

Please create a new PR with just the `tests` folder.

**Pull request closed without merging**